doc/api/v0.yaml

```diff
@@ -559,3 +575,16 @@ components:
           items:
               type: integer
               format: i64
+
+    VoterGroup:
+      properties:
+        id: integer
```

Comment:
wondering whether we could collate those two things and simply have a string identifier that could be used as a descriptive name as well, so that it's more versatile in other tools like cli utilities